app/stacks/uk-west/appeals-service/outputs.tf

```diff
@@ -33,7 +33,7 @@ output "appeal_documents_primary_blob_connection_string" {
 output "cosmosdb_connection_string" {
   description = "The connection string used to connect to the MongoDB"
   sensitive   = true
-  value       = azurerm_cosmosdb_account.appeals_database.connection_strings[0]
+  value       = azurerm_cosmosdb_account.appeals_database.primary_mongodb_connection_string

```

Comment:
this may be worth double checking https://registry.terraform.io/providers/hashicorp/azurerm/4.6.0/docs/resources/cosmosdb_account